README.md

```diff
@@ -38,6 +38,14 @@ differences:
 * `debug`: *Optional. Default `false`.* If set, progress bars will be disabled
   and debugging output will be printed instead.
 
+* `content_trust`: *Optional.* Configuration about content trust.
```

Comment:
Might want to individually note which of these sub-fields are required. I'm guessing the TLS stuff for example is optional?

Reply:
Sure, will update doc.